### ISSUE TITLE
[SUGGESTION] Is it possible to add nonum option also for endnotes?

### ISSUE THREAD

**Issue by Doc73** [maieul/ledmac#328]
This option is useful for those who, like me, use a reference system based on pstart number.

Here is a WME:
```tex
% !TeX encoding = UTF-8
% !TeX TS-program = xelatex
% !TeX spellcheck = it_IT
% rubber: set program xelatex

\documentclass[11pt,b5paper,twoside]{book}
\usepackage{fontspec}
	\setmainfont{Linux Libertine O}
\usepackage{polyglossia}

\usepackage[noeledsec,series={A,B,C}]{eledmac}
	\lineation{page}
	\linenummargin{inner}
	\sidenotemargin{outer}
	\footparagraph{C}

%\Xendlemmaseparator[A]{\rbracket}
%\renewcommand*{\printnpnum}[1]{\emph{p.~#1\newline}}
\renewcommand*{\printnpnum}[1]{}

\usepackage{hyperref}

\begin{document}
\beginnumbering
\pstart
\begin{center}
	{\Large \ledinnernote{175r}Tusculanae Disputationes}
\end{center}
\pend
\bigskip

\setcounter{pstart}{1}
\numberpstarttrue

\pstart\edlabel{itm:Rv2}%
\edtext{}{\Aendnote[nosep]{{\textbf{\pstartref{itm:Rv2}}}\enspace nota lunga nota lunga nota lunga nota lunga nota lunga nota lunga nota lunga nota lunga nota lunga nota lunga nota lunga nota lunga nota lunga nota lunga nota lunga nota lunga nota lunga nota lunga nota lunga nota lunga nota lunga nota lunga nota lunga nota lunga nota lunga nota lunga nota lunga nota lunga nota lunga nota lunga nota lunga nota lunga nota lunga nota lunga nota lunga nota lunga nota lunga nota lunga nota lunga nota lunga nota lunga nota lunga nota lunga nota lunga nota lunga nota lunga nota lunga nota lunga nota lunga nota lunga nota lunga nota lunga nota lunga nota lunga nota lunga nota lunga nota lunga nota lunga nota lunga nota lunga nota lunga nota lunga nota lunga nota lunga nota lunga nota lunga nota lunga nota lunga nota lunga nota lunga nota lunga nota lunga nota lunga nota lunga nota lunga nota lunga nota lunga nota lunga nota lunga nota lunga nota lunga nota lunga nota lunga nota lunga}}
	Cum defensionum \edtext{laboribus}{\Cfootnote{first note}} senatoriisque muneribus aut omnino aut magna ex parte essem aliquando liberatus, rettuli me, Brute, te hortante maxime ad ea studia, quae retenta animo, remissa temporibus, longo intervallo intermissa revocavi, et cum omnium artium, quae ad rectam vivendi viam pertinerent, \edtext{ratio}{\Cfootnote{second note}} et disciplina studio sapientiae, quae philosophia dicitur, contineretur, hoc mihi Latinis litteris \edtext{inlustrandum}{\Cfootnote{third note}} putavi, non quia \edtext{philosophia}{\Cfootnote{blablabla}} Graecis et litteris et doctoribus percipi non posset, sed meum semper iudicium fuit omnia nostros aut invenisse per se sapientius quam Graecos aut accepta ab illis fecisse meliora, quae quidem digna statuissent, in quibus elaborarent.
\pend

\pstart%
\ledinnernote{175v}%
Cum defensionum \edtext{laboribus}{\Cfootnote{first note}} senatoriisque muneribus aut omnino aut magna ex parte essem aliquando liberatus, rettuli me, Brute, te hortante maxime ad ea studia, quae retenta animo, remissa temporibus, longo intervallo intermissa revocavi, et cum omnium artium, quae ad rectam vivendi viam pertinerent, \edtext{ratio}{\Cfootnote{second note}} et disciplina studio sapientiae, quae philosophia dicitur, contineretur, hoc mihi Latinis litteris \edtext{inlustrandum}{\Cfootnote{third note}} putavi, non quia \edtext{philosophia}{\Cfootnote{fourth note}} Graecis et litteris et doctoribus percipi non posset, sed meum semper iudicium fuit omnia nostros aut invenisse per se sapientius quam Graecos aut accepta ab illis fecisse meliora, quae quidem digna statuissent, in quibus elaborarent.
\pend

\endnumbering
\beginnumbering
\numberpstarttrue

\pstart%
Cum defensionum \edtext{laboribus}{\Cfootnote{first note}} senatoriisque muneribus aut omnino aut magna ex parte essem aliquando liberatus, rettuli me, Brute, te hortante maxime ad ea studia, quae retenta animo, remissa temporibus, longo intervallo intermissa revocavi, et cum omnium artium, quae ad rectam vivendi viam pertinerent, \edtext{ratio}{\Cfootnote{second note}} et disciplina studio sapientiae, quae philosophia dicitur, contineretur, hoc mihi Latinis litteris \edtext{inlustrandum}{\Cfootnote{third note}} putavi, non quia \edtext{philosophia}{\Cfootnote{fourth note}} Graecis et litteris et doctoribus percipi non posset, sed meum semper iudicium fuit omnia nostros aut invenisse per se sapientius quam Graecos aut accepta ab illis fecisse meliora, quae quidem digna statuissent, in quibus elaborarent.
\pend

\pstart%
Cum defensionum \edtext{laboribus}{\Cfootnote{first note}} senatoriisque muneribus aut omnino aut magna ex parte essem aliquando liberatus, rettuli me, Brute, te hortante maxime ad ea studia, quae retenta animo, remissa temporibus, longo intervallo intermissa revocavi, et cum omnium artium, quae ad rectam vivendi viam pertinerent, \edtext{ratio}{\Cfootnote{second note}} et disciplina studio sapientiae, quae philosophia dicitur, contineretur, hoc mihi Latinis litteris \edtext{inlustrandum}{\Cfootnote{third note}} putavi, non quia \edtext{philosophia}{\Cfootnote{fourth note}} Graecis et litteris et doctoribus percipi non posset, sed meum semper iudicium fuit omnia nostros aut invenisse per se sapientius quam Graecos aut accepta ab illis fecisse meliora, quae quidem digna statuissent, in quibus elaborarent.
\pend

\pstart%
Cum defensionum \edtext{laboribus}{\Cfootnote{first note}} senatoriisque muneribus aut omnino aut magna ex parte essem aliquando liberatus, rettuli me, Brute, te hortante maxime ad ea studia, quae retenta animo, remissa temporibus, longo intervallo intermissa revocavi, et cum omnium artium, quae ad rectam vivendi viam pertinerent, \edtext{ratio}{\Cfootnote{second note}} et disciplina studio sapientiae, quae philosophia dicitur, contineretur, hoc mihi Latinis litteris \edtext{inlustrandum}{\Cfootnote{third note}} putavi, non quia \edtext{philosophia}{\Cfootnote{fourth note}} Graecis et litteris et doctoribus percipi non posset, sed meum semper iudicium fuit omnia nostros aut invenisse per se sapientius quam Graecos aut accepta ab illis fecisse meliora, quae quidem digna statuissent, in quibus elaborarent.
\pend

\pstart\edlabel{x}
\edtext{}{\Aendnote[nosep]{{\textbf{\pstartref{x}}}\enspace nota lunga nota lunga nota lunga nota lunga nota lunga nota lunga nota lunga nota lunga nota lunga nota lunga nota lunga nota lunga nota lunga nota lunga nota lunga nota lunga nota lunga nota lunga nota lunga nota lunga nota lunga nota lunga nota lunga nota lunga nota lunga nota lunga nota lunga nota lunga nota lunga nota lunga nota lunga nota lunga nota lunga nota lunga nota lunga nota lunga nota lunga nota lunga nota lunga nota lunga nota lunga nota lunga nota lunga nota lunga nota lunga nota lunga nota lunga nota lunga nota lunga nota lunga nota lunga nota lunga nota lunga nota lunga nota lunga nota lunga nota lunga nota lunga nota lunga nota lunga nota lunga nota lunga nota lunga nota lunga nota lunga nota lunga nota lunga nota lunga nota lunga nota lunga nota lunga nota lunga nota lunga nota lunga nota lunga nota lunga nota lunga nota lunga nota lunga nota lunga nota lunga nota lunga nota lunga nota lunga}}
	Cum defensionum \edtext{laboribus}{\Cfootnote{first note}} senatoriisque muneribus aut omnino aut magna ex parte essem aliquando liberatus, rettuli me, Brute, te hortante maxime ad ea studia, quae retenta animo, remissa temporibus, longo intervallo intermissa revocavi, et cum omnium artium, quae ad rectam vivendi viam pertinerent, \edtext{ratio}{\Cfootnote{second note}} et disciplina studio sapientiae, quae philosophia dicitur, contineretur, hoc mihi Latinis litteris \edtext{inlustrandum}{\Cfootnote{third note}} putavi, non quia \edtext{philosophia}{\Cfootnote{fourth note}} Graecis et litteris et doctoribus percipi non posset, sed meum semper iudicium fuit omnia nostros aut invenisse per se sapientius quam Graecos aut accepta ab illis fecisse meliora, quae quidem digna statuissent, in quibus elaborarent.
\pend

\numberpstartfalse
\endnumbering

\newpage
\doendnotes{A}
\newpage
\section{Serie A}
\doendnotesbysection{A}
\section{Serie B}
\doendnotesbysection{A}
\end{document}
```

**Comment by maieul** [comment]
you mean nonum option for a particular notee?

\Aendnote[nonum]{Content} ?

**Comment by Doc73** [comment]
Yes, right!
Very useful for long commentaries, for example.

**Comment by maieul** [comment]
ok, my question was just if you asked for global or local option.

I try to finish reledpar, then some previous issues, including one of me not open, and I will look at that

**Comment by Doc73** [comment]
Beh, both local or global, if possible. Surely, local.

**Comment by maieul** [comment]
Please try branch issue328 (which is based on reledpar). Its provides three new tools:
* [nonum] option for individual notes.
* \Xendnonumber for every notes (of a specified series, if using optional argumen)
* \Xendinplaceofnumber{<length>} which set the space inserted if the notes number are not printed.

**Comment by Doc73** [comment]
Thanka! Works very well!
If you want, you can use this sample:
```tex
% !TeX encoding = UTF-8
% !TeX TS-program = xelatex
% !TeX spellcheck = it_IT
% rubber: set program xelatex

\documentclass[11pt,b5paper,twoside]{book}
\usepackage{fontspec}
	\setmainfont{Linux Libertine O}
\usepackage{polyglossia}

\usepackage[noledgroup,noeledsec,series={A,B,C}]{reledmac}
	\lineation{page}
	\linenummargin{inner}
	\sidenotemargin{outer}
	\Xarrangement[C]{paragraph}
	\Xnonumber[A,B]
	\Xinplaceofnumber[A,B,C]{0em}
	\Xnolemmaseparator[A,B]
	\Xinplaceoflemmaseparator[A,B]{0em}
	\Xinplaceoflemmaseparator[C]{.5em}
	\Xendlemmaseparator[A]{}


%-----------------------------------------------------------------------
% TITLES FORMAT (titlesec,itnumpar)
%-----------------------------------------------------------------------
\usepackage{titlesec,itnumpar}
\titleformat{\part}[display]
	{\centering}%formatsetcount
	{\normalfont{Pars \thepart}}%label
	{30pt}%se
	{\large\MakeUppercase}%before
\titleformat{\chapter}
	{\centering}%format
	{}%label
	{00pt}%sep
	{\large\MakeUppercase}%before
\titleformat{\section}
	{\itshape}
	{\normalfont\thesection.}
	{.5em}
	{}
\titleformat{\subsection}
	{\itshape}
	{\normalfont\thesubsection.}
	{.5em}
	{}
\titlespacing*{\chapter}{0pt}{50pt}{40pt} % = default
\titlespacing*{\section}{0pt}{30pt}{10pt}
\titlespacing*{\subsection}{0pt}{30pt}{10pt}
	\renewcommand{\thepart}{\normalfont\Roman{part}}
	\renewcommand{\thechapter}{}
	\renewcommand{\thesection}{\arabic{section}}
	\renewcommand{\thesubsection}{\thesection.\arabic{subsection}}

%% STILE PAGINE
\assignpagestyle{\part}{empty}
\assignpagestyle{\chapter}{empty}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\usepackage{hyperref}

\begin{document}
\beginnumbering
\pstart
\begin{center}
	{\Large \ledinnernote{175r}Tusculanae Disputationes}
\end{center}
\pend
\bigskip

\setcounter{pstart}{1}
\numberpstarttrue

\pstart\edlabel{itm:Rv2}%
	\edtext{}{\Bfootnote{\textbf{\pstartref{itm:Rv2}}\enspace vide p.~\pageref{x1}}}
	\edtext{}{\Aendnote[nosep,nonum]{{\textbf{\pstartref{itm:Rv2}}}\enspace \label{x1}prima nota prima nota prima nota prima nota prima nota prima nota prima nota prima nota prima nota prima nota prima nota prima nota prima nota prima nota prima nota prima nota prima nota prima nota prima nota prima nota prima nota prima nota prima nota prima nota prima nota prima nota prima nota prima nota prima nota prima nota prima nota prima nota prima nota prima nota prima nota prima nota prima nota prima nota prima nota prima nota prima nota prima nota prima nota prima nota prima nota prima nota prima nota prima nota prima nota prima nota prima nota prima nota prima nota prima nota prima nota prima nota prima nota prima nota prima nota prima nota prima nota prima nota prima nota prima nota}}
	Cum defensionum \edtext{laboribus}{\Cfootnote{first note}} senatoriisque muneribus aut omnino aut magna ex parte essem aliquando liberatus, rettuli me, Brute, te hortante maxime ad ea studia, quae retenta animo, remissa temporibus, longo intervallo intermissa revocavi, et cum omnium artium, quae ad rectam vivendi viam pertinerent, \edtext{ratio}{\Cfootnote{second note}} et disciplina studio sapientiae, quae philosophia dicitur, contineretur, hoc mihi Latinis litteris \edtext{inlustrandum}{\Cfootnote{third note}} putavi, non quia \edtext{philosophia}{\Cfootnote{blablabla}} Graecis et litteris et doctoribus percipi non posset, sed meum semper iudicium fuit omnia nostros aut invenisse per se sapientius quam Graecos aut accepta ab illis fecisse meliora, quae quidem digna statuissent, in quibus elaborarent.
\pend

\pstart%
\ledinnernote{175v}%
Cum defensionum \edtext{laboribus}{\Cfootnote{first note}} senatoriisque muneribus aut omnino aut magna ex parte essem aliquando liberatus, rettuli me, Brute, te hortante maxime ad ea studia, quae retenta animo, remissa temporibus, longo intervallo intermissa revocavi, et cum omnium artium, quae ad rectam vivendi viam pertinerent, \edtext{ratio}{\Cfootnote{second note}} et disciplina studio sapientiae, quae philosophia dicitur, contineretur, hoc mihi Latinis litteris \edtext{inlustrandum}{\Cfootnote{third note}} putavi, non quia \edtext{philosophia}{\Cfootnote{fourth note}} Graecis et litteris et doctoribus percipi non posset, sed meum semper iudicium fuit omnia nostros aut invenisse per se sapientius quam Graecos aut accepta ab illis fecisse meliora, quae quidem digna statuissent, in quibus elaborarent.
\pend

\endnumbering
\beginnumbering
\numberpstarttrue

\pstart%
Cum defensionum \edtext{laboribus}{\Cfootnote{first note}} senatoriisque muneribus aut omnino aut magna ex parte essem aliquando liberatus, rettuli me, Brute, te hortante maxime ad ea studia, quae retenta animo, remissa temporibus, longo intervallo intermissa revocavi, et cum omnium artium, quae ad rectam vivendi viam pertinerent, \edtext{ratio}{\Cfootnote{second note}} et disciplina studio sapientiae, quae philosophia dicitur, contineretur, hoc mihi Latinis litteris \edtext{inlustrandum}{\Cfootnote{third note}} putavi, non quia \edtext{philosophia}{\Cfootnote{fourth note}} Graecis et litteris et doctoribus percipi non posset, sed meum semper iudicium fuit omnia nostros aut invenisse per se sapientius quam Graecos aut accepta ab illis fecisse meliora, quae quidem digna statuissent, in quibus elaborarent.
\pend

\pstart%
Cum defensionum \edtext{laboribus}{\Cfootnote{first note}} senatoriisque muneribus aut omnino aut magna ex parte essem aliquando liberatus, rettuli me, Brute, te hortante maxime ad ea studia, quae retenta animo, remissa temporibus, longo intervallo intermissa revocavi, et cum omnium artium, quae ad rectam vivendi viam pertinerent, \edtext{ratio}{\Cfootnote{second note}} et disciplina studio sapientiae, quae philosophia dicitur, contineretur, hoc mihi Latinis litteris \edtext{inlustrandum}{\Cfootnote{third note}} putavi, non quia \edtext{philosophia}{\Cfootnote{fourth note}} Graecis et litteris et doctoribus percipi non posset, sed meum semper iudicium fuit omnia nostros aut invenisse per se sapientius quam Graecos aut accepta ab illis fecisse meliora, quae quidem digna statuissent, in quibus elaborarent.
\pend

\pstart%
Cum defensionum \edtext{laboribus}{\Cfootnote{first note}} senatoriisque muneribus aut omnino aut magna ex parte essem aliquando liberatus, rettuli me, Brute, te hortante maxime ad ea studia, quae retenta animo, remissa temporibus, longo intervallo intermissa revocavi, et cum omnium artium, quae ad rectam vivendi viam pertinerent, \edtext{ratio}{\Cfootnote{second note}} et disciplina studio sapientiae, quae philosophia dicitur, contineretur, hoc mihi Latinis litteris \edtext{inlustrandum}{\Cfootnote{third note}} putavi, non quia \edtext{philosophia}{\Cfootnote{fourth note}} Graecis et litteris et doctoribus percipi non posset, sed meum semper iudicium fuit omnia nostros aut invenisse per se sapientius quam Graecos aut accepta ab illis fecisse meliora, quae quidem digna statuissent, in quibus elaborarent.
\pend

\pstart\edlabel{x}
	\edtext{}{\Bfootnote{\textbf{\pstartref{x}}\enspace vide p.~\pageref{2}}}
	\edtext{}{\Aendnote[nosep,nonum]{{\textbf{\pstartref{x}}}\enspace\label{2}seconda nota seconda nota seconda nota seconda nota seconda nota seconda nota seconda nota seconda nota seconda nota seconda nota seconda nota seconda nota seconda nota seconda nota seconda nota seconda nota seconda nota seconda nota seconda nota seconda nota seconda nota seconda nota seconda nota seconda nota seconda nota seconda nota seconda nota seconda nota seconda nota seconda nota seconda nota seconda nota seconda nota seconda nota seconda nota seconda nota seconda nota seconda nota seconda nota seconda nota seconda nota seconda nota seconda nota seconda nota seconda nota seconda nota seconda nota seconda nota seconda nota seconda nota seconda nota seconda nota seconda nota seconda nota seconda nota seconda nota seconda nota seconda nota seconda nota seconda nota seconda nota seconda nota seconda nota seconda nota seconda nota seconda nota seconda nota seconda nota seconda nota seconda nota seconda nota seconda nota seconda nota seconda nota seconda nota seconda nota}}
	Cum defensionum \edtext{laboribus}{\Cfootnote{first note}} senatoriisque muneribus aut omnino aut magna ex parte essem aliquando liberatus, rettuli me, Brute, te hortante maxime ad ea studia, quae retenta animo, remissa temporibus, longo intervallo intermissa revocavi, et cum omnium artium, quae ad rectam vivendi viam pertinerent, \edtext{ratio}{\Cfootnote{second note}} et disciplina studio sapientiae, quae philosophia dicitur, contineretur, hoc mihi Latinis litteris \edtext{inlustrandum}{\Cfootnote{third note}} putavi, non quia \edtext{philosophia}{\Cfootnote{fourth note}} Graecis et litteris et doctoribus percipi non posset, sed meum semper iudicium fuit omnia nostros aut invenisse per se sapientius quam Graecos aut accepta ab illis fecisse meliora, quae quidem digna statuissent, in quibus elaborarent.
\pend

\numberpstartfalse
\endnumbering

%\newpage
%\doendnotes{A}
\newpage
\section{Serie A}
\doendnotesbysection{A}
\section{Serie B}
\doendnotesbysection{A}
\end{document}
```

Please, correct a mistake in handbook. At pag. 32 chap. 6.3.2 marginal note is ```\Xendlemmaseparator```, but the text describes ```\Xlemmaseparator```.

**Comment by maieul** [comment]
thanks for your example. However, I would prefer a shorter, without any crossref and only endnotes.

For the mistake, the marginal note was right, it was the main content which was false.
So, corrected in a1c6962.

**Comment by maieul** [comment]
will be integrated directly in reledmac 2.0.0. I am just waiting for some correction by @ralessi to release reledmac.
